interfaces/sourcegen/sourcegen/clib/_CLibSourceGenerator.py

```diff
@@ -314,7 +334,13 @@ def _scaffold_body(self, c_func: CFunc, recipe: Recipe) -> tuple[str, set[str]]:
             template = loader.from_string(self._templates["clib-function"])
 
         elif recipe.what == "constructor":
-            template = loader.from_string(self._templates["clib-constructor"])
+            if args["handle"]:
+                raise RuntimeError()
```

Comment:
How can we end up in this state? What sort of error message would be useful to print here?

Reply:
Whoops! This was left over from some temporary checks. It is indeed not needed.